UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -176,12 +176,20 @@ public void AuthChange(ulong steamid, ulong ownerid, ServerAuth.Status status)
 			return;
 		}
 
+		//god please work, should be the leave message and associated logging.
```

Comment:
Please make your comments straight and to the point

Reply:
Fixed. My bad.